specs/lib/gltfPipelineSpec.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+var gltfPipeline = require('../../lib/gltfPipeline');
+var writeBinaryGltf = require('../../lib/writeBinaryGltf');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var gltfPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf';
+var glbPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.glb';
+
+describe('gltfPipeline', function() {
+    it('checks that gltf is defined when passing in a gltf file', function(done) {
+        gltfPipeline({
+            inputPath : gltfPath,
+            separate : false
+        }, function(gltf) {
+            expect(defined(gltf)).toEqual(true);
```

Comment:
Use `.toBeDefined()` instead. Also is there any way to make this more thorough? Maybe at least check that the gltfs don't equal each other.